functions/validator/utils.py

```diff
@@ -0,0 +1,30 @@
+__version__ = '0.0.1'
+
+import os
+import logging
+import time
+from datetime import datetime
+
+
+def set_stream_logger(name="cis-validator", level=logging.INFO,
+                      format_string=None):
+    """
```

Comment:
Often when I do this it's because there should be a docstring in the future explaining what's going on.  This is a work-in-progress.